components/core/src/clp/streaming_archive/ArchiveMetadata.hpp

```diff
@@ -102,6 +150,9 @@ class ArchiveMetadata {
     // The size of the archive
     uint64_t m_compressed_size{0};
     uint64_t m_dynamic_compressed_size{0};
+    std::string_view m_variable_encoding_methods_version{ffi::cVariableEncodingMethodsVersion};
+    std::string_view m_variables_schema_version{ffi::cVariablesSchemaVersion};
+    std::string_view m_compression_type{cCompressionTypeZstd};
```

Comment:
Can we add these in another PR? Switching to MessagePack and adding the new fields are kind of unrelated changes.